### ISSUE TITLE
ProPresenter won't connect

### ISSUE THREAD

**Issue by ScottGlascock** [bitfocus/companion#882]
**Describe the bug**
ProPresenter module not functional. Will not connect (Status never changes to OK)

**To Reproduce**
Make a button to control ProPresenter. When you push a button, red error square on stream deck surface lights up, "error" shows up in instance status. The buttons I've been creating/testing are "Next Slide" and "Clear All Slides"

**Comment by cxselph** [comment]
Have you followed the instructions in the help file?  https://github.com/bitfocus/companion-module-renewedvision-propresenter/blob/fb56dbd3f25c2437767f45af58290b6f5e01dd5b/HELP.md

**Comment by ScottGlascock** [comment]
Yes. I also double checked by using the same settings in version 1.4 and it connected instantly.

**Comment by Stew-Smith** [comment]
+1 verifying this issue. Problem exists in both Windows and PC environments.  the instance will show either no status or ok status until you try to fire a command then it shows disconnected.  not showing any errors in the logs.
